modules/grafana/local/datasource_credentials.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+#
+# Set the credentials for cortex and loki datasources as environment variables to be used in provisioning/datasources/automatic.yml
+for s in $(aws secretsmanager get-secret-value --secret-id sre/monitoring/all-loki-credentials --query SecretString --output text --region us-east-2 | jq -r "to_entries|map(\"\(.key)=\(.value|tostring)\")|.[]"); do
```

Comment:
I would make the secret path an environment variable and pass it in - again making it more useful as a public repo.

Reply:
Moved this to https://gitlab.com/mintel/satoshi/tools/build-harness-extentions-private. I think someone using this opensource would have to implement their own solution to grabbing secrets, they would have it set up differently than we do so this script would not be useful to them.